### DISCUSSION TITLE
Cannot run properly the basic scenario in Scenario Runner

### DISCUSSION THREAD

**Discussion by malak-peter** [carla-simulator/carla#4276] [Q&A]
Hi,
I followed the instructions mentioned in the doc of CARLA to install the ScenarioRunner. After I set the CARLA_ROOT, SCENARIO_RUNNER_ROOT and PYTHONPATH I run the script:
py -3.7 scenario_runner.py --scenario FollowLeadingVehicle_1 --reloadWorld
The scenario is loading, but there is only the ego vehicle on the road, no leading vehicle. After 1 min, the scenario is over and I get this message:
![image](https://user-images.githubusercontent.com/85319341/120720484-7037a700-c4cc-11eb-9dc7-9a76f38af53c.png)

In addition: if I open another terminal and use the manual control (py -3.7 manual_control.py) while the previous scenario is running, I get the following error:

Traceback (most recent call last):
File “manual_control.py”, line 58, in
from examples.manual_control import (World, HUD, KeyboardControl, CameraManager,
ModuleNotFoundError: No module named ‘examples.manual_control’

This second one is the main problem.

Please provide information on these errors. Thank you in advance!

**Comment by corkyw10** [comment]
Hi @malak-peter, when you load the scenario, the ego vehicle appears, but the leading vehicle will only appear once the ego starts to drive. Did the leading vehicle not appear once your ego was moving?

For the error on running `manual_control.py`, can you try adding `${CARLA_ROOT}/PythonAPI/` to your `PYTHONPATH` if you haven't already got it?

**Reply by udaydeloitte** [reply]
getting same error.
